projects/imminence/docker-compose.yml

```diff
@@ -17,28 +18,39 @@ x-imminence: &imminence
   working_dir: /govuk/imminence
 
 services:
+  imminence-pg:
+    image: ghcr.io/baosystems/postgis:14-3.2
```

Comment:
Do we need to use github container registry for this? I'm somewhat guessing you've had to go this way to get arm64 and amd64 containers but not sure. I think all our other images are dockerhub ones currently.

Reply:
Hmm. At the time I was initially making this repo, this was the only one available that would work. But it may be that it's changed now. I'll check what's available - it's this version of the image that requires the password, too, so it's possible that if we change it we might not need the password.

Reply:
The official Postgis container only comes in AMD64 versions, so is unusable for local dev on M1 machines. I've replaced this with a dockerhub container that seems reasonably up-to-date, but it's "just some person". I guess that's probably not too bad a thing, given that most of the open source packages we use are just some person, but I don't know if other people might have a more solid opinion than mine, which is just "eh... it's not ideal, but I guess it's okay?"

Reply:
https://hub.docker.com/r/nickblah/postgis

Reply:
Yeah I think it's in the realm of "eh... it's not ideal, but I guess it's okay?"

I'd add a comment above it explaining we're not using official one because of lack of AMD64 support.

I notice there is https://hub.docker.com/r/kartoza/postgis which seems to have a couple of orders magnitude more downloads and AMD64 images (but may have missed something that makes it unsuitable)

Reply:
Not sure why I didn't use that one, I saw it, my brain just glossed over it. But yeah, that also works and seems more stable, so I've updated to use that one.